### PR TITLE
V3: Slows component minimization to be closer to V2

### PR DESCRIPTION
**Pull request by eireland**

### CODE REVIEW

File: v3/src/components/free-tile-row.scss [hunk 1]
```diff
@@ -10,7 +10,7 @@
     position: absolute;
     box-shadow: 0 1px 3px 0 rgb(0 0 0 / 25%);
     border-radius: 6px;
-    transition: height 0.25s;
+    transition: height 0.5s ease-in-out 0s;
```
Comment (bfinzer) [inline v3/src/components/free-tile-row.scss:13]: In V2 the duration of the transition is 0.4s. Unless there's a reason, let's use that.

**Review by bfinzer — APPROVED** [review]
👍🏻LGTM
* Just suggestion to use same value as in V2 if that seems right.

File: v3/src/components/free-tile-row.scss (hunk 1 above)
Comment (kswenson) [reply v3/src/components/free-tile-row.scss:13]: The slider has its own `transition` statement. Whatever values are decided upon, it should probably be put in `vars.scss` so it's not duplicated in multiple places.

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Looks good -- just a suggestion to DRY the transition definition.

**Review by kswenson — APPROVED** [review]
👍 LGTM

**Pull request merged**